### DISCUSSION TITLE
PocketMRTH - A Mobile App Companion for monitoring Mirth Connect Instances

### DISCUSSION THREAD

**Discussion by litemikx** [nextgenhealthcare/connect#5925] [Show and tell]
Not sure if this is the best place to share this, but I recently released a mobile app called PocketMRTH. This allows you to monitor multiple Mirth Connect instances and its channel statistics. The only language I'm a bit "proficient" is JavaScript since Mirth Connect allows you to script inside the transformers and filters, so I built this app using React Native and Expo.dev. 

You can learn more about this project here: https://github.com/litemikx/pocketmrth 

It's in Early Access so feel free to send any feedback or questions. :)

**Comment by rjkroll** [comment]
Going to check it out, thanks for sharing.  I was building a little side-project using Dash and Python to hit the Client API and return things, so interested in seeing what you've done.

**Reply by litemikx** [reply]
It's basic functionality and focused on visualized charts, just pulling data from the MC Client API. But might add advanced commands like stop/start channels in the future. :)
